bootstrap/terraform/main.tf

```diff
@@ -43,10 +43,11 @@ data "aws_caller_identity" "current" {}
 data "aws_availability_zones" "available" {}
 
 locals {
-  name   = var.name
-  region = var.region
+  name   = "crossplane-blueprints"
+  region = "us-east-1"
 
-  cluster_version = var.cluster_version
+  cluster_version = "1.30"
+  capacity_type   = "SPOT"
```

Comment:
same question, what is the reason for removing the vars?

Reply:
the same here:

- [version](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/3bfc0f072e908d6f3911c247edaeb429b8e429b2/patterns/ipv6-eks-cluster/main.tf#L29).

Reply:
same here, keep the vars

Reply:
done